File: Minimize Sum Of Array (Array Series #1).dart
int minSum(List<int> arr) {
  arr.sort();
  var sum = 0;
  for (var left = 0, right = arr.length - 1;left < arr.length / 2;left++, right--) {
    sum += arr[left] * arr[right];
  }
  return sum;
}

File: Minimize Sum Of Array (Array Series #1)/Minimize Sum Of Array (Array Series #1).js
function minSum(arr) {
  let arr_s = arr.slice().sort((a, b) => a - b);
  let result = 0;
  for (let i = 0; i < Math.floor(arr_s.length / 2); i++) {
    result += arr_s[i] * arr_s[arr_s.length - 1 - i];
  }
  return result;
}
